test/test.ml

```diff
@@ -85,20 +101,35 @@ let bad_test f file () =
 
 let excl_test file () =
   Alcotest.(check bool) "good test" true true
+module P4 = Types 
+let pp_test () =
+  Alcotest.(check bool) "printing test" true Printingtests.test
+
+let example_path l =
+  let root = Filename.concat ".." "examples" in
+  List.fold_left l ~init:root ~f:Filename.concat
 
-let () =
+let () = 
+  (* Format.printf "%s" (get_name ["../examples"] (example_path ["checker_tests"; "good"; "coree.p4"])); 
+     Format.printf "%s" "----------";
+     Format.printf "%s" ((Parse.parse_string (get_name ["../examples"] (example_path ["checker_tests"; "good"; "coree.p4"]))) |> Prettypp.format_program |> to_string);   *)
   let open Alcotest in
   run "Tests" [
-    "excluded tests good", (Stdlib.List.map (fun name ->
+    (* "excluded tests good", (Stdlib.List.map (fun name ->
         test_case name `Quick (excl_test name)) excluded_good_files);
```

Comment:
Let's uncomment the other tests. You can run just the pp tests by running `dune exec -- ./test.exe test pp` from inside the `test/` folder. The last argument there, `pp`, can be any regular expression you want that matches "round trip pp tests good" (or whatever tests you want to run).